package.json

```diff
@@ -7,11 +7,12 @@
     "@react-navigation/stack": "^5.9.3",
     "@reduxjs/toolkit": "^1.4.0",
     "@shopify/restyle": "^1.3.0",
+    "@types/jest": "^26.0.15",
     "expo": "~39.0.0",
     "expo-splash-screen": "~0.6.2",
     "expo-status-bar": "~1.0.2",
     "expo-updates": "~0.3.2",
-    "react": "16.13.1",
+    "react": "^16.14.0",
```

Comment:
Q: Are we sure we can keep the `react` version decoupled from the `react-native` version?

Reply:
Yes, it should be fine. `16.14.0` Only added the JSX package and a couple bug fixes :)